conduct.md

```diff
@@ -4,66 +4,85 @@ layout: page
 
-### Reporting harassment
+### What can happen if the CoC is violated?
 
-If you are being harassed, notice that someone else is being harassed, or have any other concerns,
-please contact the Conduct team immediately either in person, or by sending an email to
+In case of a CoC violation, some of the most common actions organisers may take are:
+
+* Demanding that a participant stops their behaviour.
+* Demanding that a participant prevents further contact with certain other participants.
+* Cancelling a conference talk.
+* Removing a participant from the conference, without refund.
+
+The action taken is at the discretion of the Code of Conduct team. Participants are expected to comply immediately, and further action may be taken in case a participant does not comply. A record will be kept of all incidents. 
+
+### Where to report incidents
+
+If a Code of Conduct incident happens to you, or witness it happening to someone else, please contact the Code of Conduct team immediately either in person, or by sending an email to
 conduct@djangocon.eu.
 
-If the matter is especially urgent, please notify DjangoCon Europe staff via
-[email](mailto:conduct@djangocon.eu). We will provide more instant points of contact (Slack, phone)
-once we are closer to the conference.
+Code of Conduct team members will be introduced in person at the conference. You can approach any of them at any time to discuss an incident. We will also post their names closer to the conference. You can also ask any other staff to help you find them at the conference. We will provide more instant points of contact (Slack, phone) once we are closer to the conference.
+
+### Guidelines for reporting incidents
+
+Please do not feel like you may be a burden to us by reporting incidents. Even if you happen to report multiple incidents during the conference. We rather consider reports an opportunity for us to act: by knowing about an incident, we can act on it, and often prevent it from continuing or repeating. But if we don’t know, we can’t take action.
+
+If you are not sure whether the situation was a Code of Conduct violation, or whether it applied to that particular space, we encourage you to still report it. We would much rather have a few extra reports where we decide to take no action, rather than miss a report of an actual violation. We do not look negatively on you if we find the incident is not a violation. And knowing about incidents that are not violations, or happen outside our spaces, can also help us to improve the Code of Conduct or the processes surrounding it.
 
-If you’re currently afraid of your physical safety or are in danger, contact local law enforcement
-in Germany:
+In your report please include, when possible:
+
+* Your contact info (so we can get in touch with you)
```

Comment:
"If you feel unsafe reporting in person, you may choose one person to represent you. In this case, we'd need their contact information, but we'd ask you to make clear that this person represents you." ?

Reply:
👍